charts/bridgeai-prediction-service/values.yaml

```diff
@@ -0,0 +1,126 @@
+# Default values for bridgeai-prediction-service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/digicatapult/bridgeai-prediction-service
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.1.0"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+modelPredictionEndpoint: "http://model/prediction/endpoint"
```

Comment:
Are we mentioning somewhere in the readme that this need to be updated with proper endpoint of the model?

Reply:
We are not, but we should, you are correct.